tests/scripts/t3000/run_t3000_demo_tests.sh

```diff
@@ -31,7 +31,7 @@ run_t3000_llama3_70b_tests() {
   echo "LOG_METAL: Running run_t3000_llama3_70b_tests"
 
   # Llama3 70B demo (output verification)
-  env WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest models/demos/t3000/llama3_70b/demo/demo.py::test_LlamaModel_demo[wormhole_b0-True-check_enabled-greedy-tt-70b-T3000-80L-decode_only-text_completion-llama3] --timeout=900
+  env WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest models/demos/t3000/llama3_70b/demo/demo.py::test_LlamaModel_demo[wormhole_b0-True-short_context-check_enabled-greedy-tt-70b-T3000-80L-decode_only-text_completion-llama3] --timeout=900
```

Comment:
there's a movement to cut down on directly invoking specific pytests in test files and to invoke the .py file directly (without **.py::[])

can we follow this convention here?
Looking at demo.py there doesn't seem to be other tests so I think this is a trivial change.
Though the README.md will need to be updated as well to match invoking this pytest in particular

Reply:
We need to leave all those options available so we/users can run the demo with a variety of args.

Can we open a separate issue so we can create a `demo_t3000.py` test file just like we've done for all the other pytests?

Reply:
sounds good

Reply:
You can assign it to me

Reply:
@ttmchiou is this alright to merge?

Reply:
https://github.com/tenstorrent/tt-metal/issues/10625
for reference